x-pack/test/accessibility/apps/group3/ml_embeddables_in_dashboard.ts

```diff
@@ -98,6 +97,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         it('can open job selection flyout', async () => {
           await PageObjects.dashboard.clickCreateDashboardPrompt();
           await ml.dashboardEmbeddables.assertDashboardIsEmpty();
+          // FIXME remove sleep when https://github.com/elastic/kibana/issues/187587 if fixed
+          await PageObjects.common.sleep(3000);
```

Comment:
Had to add `sleep` due to [this bug](https://github.com/elastic/kibana/issues/187587). When it's fixed, we should replace this hardcoded delay and wait for the list initializaiton.